node/external/interface.go

```diff
@@ -1,8 +1,16 @@
 package external
 
+import (
+	"github.com/ElrondNetwork/elrond-go/process/smartContract"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+)
+
+// TODO: Move interface to scDataGetter.go
```

Comment:
interface has to remain where it is used

Reply:
@andreibancioiu The location of interfaces has been discussed at length, long ago. By convention, they remain where they are used. I forgot why. @iulianpascalau might remember why. I still disagree, but will not reopen the topic. You can leave the interface here.